datalad/support/annexrepo.py

```diff
@@ -3661,7 +3661,11 @@ def close(self, return_stderr=False):
 
 def _get_size_from_perc_complete(count, perc):
     """A helper to get full size if know % and corresponding size"""
-    portion = (float(perc) / 100.)
+
+    try:
+        portion = (float(perc) / 100.)
+    except ValueError:
+        return None
```

Comment:
ATM it should be ok to introduce `None` as possible output here (at the use point it is `or`-ed with 0 after all), but I guess it would have been safer to just return 0 (although `None` is more semantically correct).
I am just afraid that if above code is changed under assumption that this one returns an `int`, we could be in trouble without realizing it (since there is no dedicated "integration" test for this use case). 
So - I am ok with it as is, but ideally - should we just return 0 here (or just `portion = None` and leave return to logic below)